### PR TITLE
terraform-docs 0.1.0 (new formula)

### PR DESCRIPTION
**Pull request by jacobwgillespie**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

This PR adds the `terraform-docs` CLI tool from https://github.com/segmentio/terraform-docs. 🎉

### CODE REVIEW

File: Formula/terraform-docs.rb [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  end
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/segmentio/terraform-docs").install buildpath.children
+    Language::Go.stage_deps resources, buildpath/"src"
+    cd "src/github.com/segmentio/terraform-docs" do
+      system "go", "build", "-o", "terraform-docs"
+
+      bin.install "terraform-docs"
```
Comment (bfontaine) [inline Formula/terraform-docs.rb]: Please use `system "go", "build", "-o", "#{bin}/terraform-docs"` so you can remove this line.

File: Formula/terraform-docs.rb [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    minimal = testpath/"main.tf"
+    minimal.write <<-EOS.undent
```
Comment (bfontaine) [inline Formula/terraform-docs.rb]: You probably don’t need to create a variable if you use it once:
```ruby
(testpath/"main.tf").write <<-EOS.undent
  ...
```

**Review by bfontaine — CHANGES_REQUESTED** [review]

**Comment by jacobwgillespie** [comment]
Good catches, updated.

**Review by bfontaine — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks a lot @jacobwgillespie for your contribution to Homebrew!

**Pull request closed without merging**